### PR TITLE
AC: Added support for ssd-resnet-34-tf model

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/ssd.py [hunk 1]
```diff
@@ -449,3 +449,110 @@ def process(self, raw, identifiers, frame_meta):
...
+        return iou
+
+    def process(self, raw, identifiers, frame_meta):
+        result = []
+        raw_output = self._extract_predictions(raw, frame_meta)
+        if not self.outputs_verified:
+            self.select_output_blob(raw_output)
+
+        for identifier, logits, boxes in zip(identifiers, raw_output[self.scores_out], raw_output[self.boxes_out]):
+            detections = {'labels': [], 'scores': [], 'x_mins': [], 'y_mins': [], 'x_maxs': [], 'y_maxs': []}
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/ssd.py:506]: looks like this structure is useless (you obtain boxes, labels and scores once for single image, no need to repack them to dictionary. you can directly use variables for prediction class initialization

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/ssd.py [hunk 2]
```diff
@@ -449,3 +449,110 @@ def process(self, raw, identifiers, frame_meta):
...
+            detections['labels'].extend(labels)
+            detections['scores'].extend(probs)
+            detections['x_mins'].extend(x_mins)
+            detections['y_mins'].extend(y_mins)
+            detections['x_maxs'].extend(x_maxs)
+            detections['y_maxs'].extend(y_maxs)
+
+            result.append(DetectionPrediction(identifier, detections['labels'], detections['scores'],
+                                              detections['x_mins'], detections['y_mins'], detections['x_maxs'],
+                                              detections['y_maxs']))
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/ssd.py:520]:
```suggestion
            locs, labels, probs = self.decode_single(boxes, logits, self.iou_threshold, self.keep_top_k)

            y_mins, x_mins, y_maxs, x_maxs = locs.T

            result.append(DetectionPrediction(identifier, labels, scores, x_mins, y_mins, x_maxs, y_maxs))
```

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/ssd.py [hunk 3]
```diff
@@ -449,3 +449,110 @@ def process(self, raw, identifiers, frame_meta):
                                               detections['x_mins'], detections['y_mins'], detections['x_maxs'],
                                               detections['y_maxs']))
         return result
+
+
+class SSDTFAdapter(Adapter):
+    __provider__ = 'ssd_tf'
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/ssd.py:455]: what the main difference between this adapter and ssd_multilabel?
I see that they look very similar, except for filtering background label.
Is it possible to combine these adapters into one?

Comment (Anna-Mironova) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/ssd.py]: Done

File: tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md [hunk 4]
```diff
@@ -489,4 +489,6 @@ AccuracyChecker supports following set of adapters:
   * `boxes_out` - name of output layer with bounding boxes coordinates.
   * `confidence_threshold` - lower bound for valid boxes scores (optional, default 0.01).
   * `nms_threshold` - overlap threshold for NMS (optional, default 0.45).
-  * `keep_top_k ` - maximal number of boxes which should be kept (optional, default 200).
+  * `keep_top_k` - maximal number of boxes which should be kept during NMS (optional, default 200).
+  * `diff_coord_order` - ordering convention of coordinates differs from standard (optional, default False).
```
Comment (eaidova) [inline tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md:493]: could you please explain what is standard and different in this case?

Comment (Anna-Mironova) [reply tools/accuracy_checker/openvino/tools/accuracy_checker/adapters/README.md]: Done

**Review by eaidova — APPROVED** [review]
